code/datums/mood_events/generic_positive_events.dm

```diff
@@ -333,9 +333,23 @@
 
 /datum/mood_event/fishing
 	description = "Fishing is relaxing."
-	mood_change = 5
+	mood_change = 4
 	timeout = 3 MINUTES
 
+/datum/mood_event/fish_released
+	description = "Go, fish, swim and be free!"
+	mood_change = 1
+	timeout = 2 MINUTES
+
+/datum/mood_event/fish_released/add_effects(morbid, obj/item/fish/fish)
+	if(!morbid)
+		description = "Go, [fish.name], swim and be free!"
+		return
+	if(fish.status == FISH_DEAD)
+		description = "Releasing dead fish is relaxing."
+	else
+		description = "Go, [fish.name], swim and die soon!"
```

Comment:
```suggestion
	if(fish.status == FISH_DEAD)
		description = "Some scavenger will surely find a use for the remains of [fish.name]. How pragmatic."
	else
		description = "Returned to the burden of the deep. But is this truly a mercy, [fish.name]? There will always be bigger fish..."
```

I kind of wrote most of the morbid text as...overly analytical and verbose.

Reply:
Thank you! My mind was blank and uninspired when I coded this.